### DISCUSSION TITLE
unban.txt in user folder not being cleared

### DISCUSSION THREAD

**Discussion by bussuite** [DigitalRuby/IPBan#255] [General]
I wrote a program to allow a user to unban themselves from the list if they get added accidentally.

Since the user can't edit the firewall rule, I did it using the unban.txt method.

A user doesn't have rights to the c:\program files folder, so the file is created in the "C:\Users\<username>\AppData\Local\VirtualStore\Program Files\IPBan" folder.

Is there any way to integrate this behaviour into the update cycle?

**Comment by jjxtra** [comment]
Not really, this would let a non-admin user render the software useless. So you would need something that could move the file to the install folder. Or put the install folder in a non-admin folder.

**Comment by bussuite** [comment]
Thanks, I ended up having to give the person rights to the full IPBan folder, which is more problematic as it gives them full configurability of the software rather than just the specific unban.txt function. 

Would you consider adding a new option to specify an additional folder just for the unban.txt and ban.txt files? That way it doesn't change the out of the box behaviour but offers an alternative to full folder access?

**Reply by jjxtra** [reply]
I will look into it.
